tailcfg/tailcfg.go

```diff
@@ -1018,7 +1017,7 @@ func (req *RegisterRequest) Clone() *RegisterRequest {
 	}
 	res.DeviceCert = append(res.DeviceCert[:0:0], res.DeviceCert...)
 	res.Signature = append(res.Signature[:0:0], res.Signature...)
-	res.NodeKeySignature = append(res.NodeKeySignature[:0:0], res.NodeKeySignature...)
+	res.NodeKeySignature = res.NodeKeySignature
```

Comment:
hah, this is still manually done. sent https://github.com/tailscale/tailscale/pull/8973